hack/release-tools/tag-release.sh

```diff
@@ -28,9 +28,18 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 function tag_and_push() {
     echo "Tagging and pushing $VELERO_VERSION"
```

Comment:
This comment could be split so that the "pushing" part of it is only printed when `$publish` is true.